timemachine/cpp/src/kernels/k_rotations.cu

```diff
@@ -112,9 +113,9 @@ void __global__ k_rotate_and_translate_mols(
         int mol_end = mol_offsets[mol_sample + 1];
         int num_atoms = mol_end - mol_start;
 
-        RealType translation_x = box_x * translations[sample_idx * 3 + 0];
-        RealType translation_y = box_y * translations[sample_idx * 3 + 1];
-        RealType translation_z = box_z * translations[sample_idx * 3 + 2];
+        RealType translation_x = SCALE ? box_x * translations[sample_idx * 3 + 0] : translations[sample_idx * 3 + 0];
```

Comment:
not sure what this SCALE is doing?

Reply:
Will add a docstring, but it is simply whether to use the translation as is (such as generating inner or outer translations) or to scale the translation translations by the box dimensions (uniform unit vector in the case of biased deletion).

Reply:
Added docstring in https://github.com/proteneer/timemachine/commit/c404955a3b250e1499d377598b38db85eb6cf6c5